app/styles/pages/public-event.scss

```diff
@@ -92,3 +92,12 @@
     margin-top: 10px;
   }
 }
+
+.loc-map{
+  height: 100%;
+}
+
+.g-map-canvas {
```

Comment:
Please scope this such that this is used only for event map and not for all other g-map instances